actions/receive/action.yaml

```diff
@@ -1,6 +1,10 @@
 name: 'receive'
 description: 'Action to run {touchstone} benchmarks and upload the results.'
 inputs:
+  force_upstream:
+    description: 'Always compare against the upstream base branch when benchmarking PRs in forked repositories.'
+    required: false
+    default: false
```

Comment:
In `use_touchstone()` and `use_touchstone_workflow()`, we have argument `force` defaulting to `TRUE`. Is that an inconsistency with this default?

Reply:
The default in the action is `false` as to not cause changed behaviour with existing workflows (that's not a lot so we could still change it) that don't set it but for `use_touchstone*` (-> new workflows) I set it to `true` because I think that makes more sense in the spirit of the package. ( You want an accurate Benchmark when preparing a PR to upstream)

Reply:
Makes sense.